index.html

```diff
@@ -3,18 +3,35 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>TV Show Project | My Name (My GitHub username)</title>
+    <title>TV Show Project | Shimen (ShimenAfshar)</title>
     <link href="style.css" rel="stylesheet" />
     <link rel="icon" href="/favicon.ico" type="image/x-icon" />
   </head>
-
   <body>
-    <div id="root">
-    </div>
+   
+   <!-- search bar container -->
+    <header>
+      <a href="#"><h1>Movies</h1></a>
+      <form id="form">
+        <input type="text" class="search" placeholder="Search" id="search">
+        <label for="movies">Choose your Movie </label>
+        
+        <select name="movies" id="movies" >
+          <option selected></option>
```

Comment:
You could add a default value here of something like `Show All `(you would then need to add an `if` statement in your select listener to show all if this value was selected)